### ISSUE TITLE
[Accessibility]: Tree page: Narrow viewport/high zoom issues

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35296]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34422

### Problem description

1. Overly difficult to access action items on a small screen
<img width="484" alt="image" src="https://user-images.githubusercontent.com/9516420/167855424-5b68189e-8b7b-45b7-9d83-bf1d274df04b.png">

2. Overly difficult to read page tabs on a small screen
<img width="310" alt="image" src="https://user-images.githubusercontent.com/9516420/167855885-20f83be0-e133-4946-9967-34d37b248cfa.png">

### Expected behavior

Recommended fixes (mainly through `flex-direction: column`):

1. Break the action items on a new line, and list vertically.

2. Break the page tabs onto a new line, and read the list vertically

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
Yep, duplicate of #35239.
